frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -39,6 +39,27 @@ pods:
           server-properties:
             template: "{{CONFIG_TEMPLATE_PATH}}/server.properties.mustache"
             dest: "{{KAFKA_VERSION_PATH}}/config/server.properties"
+        readiness-check:
+          cmd: |
+            # since the server is just started, the starting and started log lines
```

Comment:
This is ~20 lines of bash (~ 1/2 of it comments).

Why is the overhead of adding it as an asset worth it?